Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM fedora:32
+RUN dnf -y update
+RUN dnf -y install git
+RUN dnf -y groupinstall "Development Tools" "Development Libraries"
+RUN git clone https://github.com/facebookincubator/cinder.git
```

Comment:
This should be changed to `ADD . /cinder`

Reply:
LGTM